src/homepageExperience/components/steps/cli/InstallDependencies.tsx

```diff
@@ -0,0 +1,312 @@
+  BorderType,
+} from '@influxdata/clockface'
+
+import React, {useEffect, useState} from 'react'
+import {useDispatch} from 'react-redux'
+import {getBuckets} from 'src/buckets/actions/thunks'
+import {event} from 'src/cloud/utils/reporting'
+
+export const InstallDependencies = () => {
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    dispatch(getBuckets())
+  }, [])
+
+  const logCopyCodeSnippetMac = () => {
+    event('firstMile.cliWizard.installDependenciesMac.code.copied')
+  }
+  const logCopyCodeSnippetWindows = () => {
+    event('firstMile.cliWizard.installDependenciesWindows.code.copied')
+  }
+  const logCopyCodeSnippetLinux = () => {
+    event('firstMile.cliWizard.installDependenciesLinux.code.copied')
+  }
+
+  const windowsCodeSnippet =
+    "> Expand-Archive .\\influxdb2-client-latest-amd64.zip -DestinationPath 'C:\\Program Files\\InfluxData\\'\n" +
+    "> mv 'C:\\Program Files\\InfluxData\\influxdb2-client-latest-windows-amd64' 'C:\\Program Files\\InfluxData\\influx'"
+  const linuxCodeSnippetA = `# amd64
+wget https://dl.influxdata.com/influxdb/releases/influxdb2-client-latest-linux-amd64.tar.gz
+  
+# arm
+wget https://dl.influxdata.com/influxdb/releases/influxdb2-client-latest-linux-arm64.tar.gz
+  `
+  const linuxCodeSnippetB = `# amd64
+tar xvzf path/to/influxdb2-client-latest-linux-amd64.tar.gz
+  
+# arm
+tar xvzf path/to/influxdb2-client-latest-linux-arm64.tar.gz
+  `
+  const linuxCodeSnippetC = `# amd64
+sudo cp influxdb2-client-latest-linux-amd64/influx /usr/local/bin/
+            
+# arm
+sudo cp influxdb2-client-latest-linux-arm64/influx /usr/local/bin/
+  `
+
+  const [macSelected, setMacSelected] = useState(true)
+  const [windowsSelected, setWindowsSelected] = useState(false)
+  const [linuxSelected, setLinuxSelected] = useState(false)
+
+  return (
+    <>
+      <h1>Install Dependencies</h1>
+      <ButtonGroup orientation={Orientation.Horizontal}>
+        <Button
+          text="mac OS"
+          color={macSelected ? ComponentColor.Primary : ComponentColor.Default}
+          onClick={() => {
+            setMacSelected(true)
+            setWindowsSelected(false)
+            setLinuxSelected(false)
+          }}
+        />
+        <Button
+          text="windows"
+          color={
+            windowsSelected ? ComponentColor.Primary : ComponentColor.Default
+          }
+          onClick={() => {
+            setMacSelected(false)
+            setWindowsSelected(true)
+            setLinuxSelected(false)
+          }}
+        />
+        <Button
+          text="linux"
+          color={
+            linuxSelected ? ComponentColor.Primary : ComponentColor.Default
+          }
+          onClick={() => {
+            setMacSelected(false)
+            setWindowsSelected(false)
+            setLinuxSelected(true)
+          }}
+        />
+      </ButtonGroup>
+      {macSelected && (
+        <>
+          <h2 style={{marginTop: '48px', marginBottom: '0px'}}>Use Homebrew</h2>
```

Comment:
since this style object literal is used in multiple places, it's probably best to pull it out into a variable that is defined *outside* the function component, so it's not redefined on every render call.

e.g.
```ts
const headingWithMargin = {marginTop: '48px', marginBottom: '0px'}
...
<h2 style={headingWithMargin}
```
the variable name is just an example. It doesn't need to be named `headingWithMargin`